shuup/admin/modules/shops/forms.py

```diff
@@ -26,9 +26,8 @@ class Meta:
     def __init__(self, **kwargs):
         super(ShopBaseForm, self).__init__(**kwargs)
         self.fields["logo"].widget = MediaChoiceWidget(clearable=True)
-        locale = get_current_babel_locale()
         self.fields["currency"] = forms.ChoiceField(
-            choices=sorted(locale.currencies.items()),
+            choices=sorted([(currency.code, currency) for currency in Currency.objects.all()]),
```

Comment:
maybe use `choices=[(currency.code, currency) for currency in Currency.objects.all().order_by("translations__name")]` or `"code"` but no need to run `sorted` here imo